bundles.rst

```diff
@@ -86,26 +105,24 @@ The directory structure of a bundle is meant to help to keep code consistent
 between all Symfony bundles. It follows a set of conventions, but is flexible
 to be adjusted if needed:
 
-``Controller/``
+``src/Controller/``
     Contains the controllers of the bundle (e.g. ``RandomController.php``).
 
-``DependencyInjection/``
```

Comment:
I propose to remove this entry from this very first moment, as the current text says, it's not necessary. After, in another doc we're mentioning this dir for extension convention